FAQ about how to completely delete a host (#87)
harvester/docs: docs/host/host.md

![cordon-node.png](./assets/cordon-nodes.png)

## Deleting a Node

> To completely delete a node, you need to
> 
> 1. Go to the **Hosts** page.
> 2. On the node you want to modify, click **⋮ > Delete**.
> 3. Login to that node as root. Then run `rke2-uninstall.sh` to delete the whole RKE2 service.

Reviewer: ```suggestion
Deleting a node is done in two phases:

1. Delete the node from Harvester
    - Go to the **Hosts** page
    - On the node you want to modify, click **⋮ > Delete**

2. Uninstall RKE2 from the node
    - Login to the node as root
    - Run `rke2-uninstall.sh` to delete the whole RKE2 service.
```

!!! warning

>     You will lost all data of the control plane on the node after deleing RKE2 service.

Reviewer 2: - \s\lost\lose
- \s\RKE2\the RKE2

>     You will lose all data of the control plane on the node after deleing the RKE2 service.

Reviewer: ```suggestion
    You will lose all data of the control plane node after deleing the RKE2 service.
```

!!! note

>     This is a [known issue](https://github.com/harvester/harvester/issues/1497) about node hard delete.
>     Once it is resolved. The last step can be skipped.

Reviewer: ```suggestion
    There's a [known issue](https://github.com/harvester/harvester/issues/1497) about node hard delete.
    Once resolved, the last step can be skipped.
```
